src/services/services.ts

```diff
@@ -7866,6 +7886,30 @@ namespace ts {
             return [];
         }
 
+        function getCodeFixesAtPosition(fileName: string, start: number, end: number, errorCodes: string[]): CodeAction[] {
+            synchronizeHostData();
+            const sourceFile = getValidSourceFile(fileName);
+            const checker = program.getTypeChecker();
+            let allFixes: CodeAction[] = [];
+
+            forEach(errorCodes, error => {
+                const context = {
+                    errorCode: error,
+                    sourceFile: sourceFile,
+                    span: { start, length: end - start },
+                    checker: checker,

```

Comment:
this should be program and not checker. since you can get the checker from the program but not the opposite.